ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/OpenSSL/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+###########
+# OpenSSL #
+###########
+- name: Check if OpenSSL 32bit installed
+  win_stat:
+    path: 'C:\openjdk\OpenSSL-1.1.1-x86_32'
+  register: openssl32_installed
+  tags: openssl
+
+- name: Check if OpenSSL 64bit installed
+  win_stat:
+    path: 'C:\openjdk\OpenSSL-1.1.1-x86_64'
+  register: openssl64_installed
+  tags: openssl
+
+- name: Check if OpenSSL downloaded
```

Comment:
If its not installed, determined by the previous check, do everything.  I don't think this is needed.